core/scripts/run-tests.sh

```diff
@@ -191,8 +191,13 @@ if ($args['cache']) {
   simpletest_script_init(NULL);
 
   echo "\nPreparing database and configuration cache for profiles\n";
+  $skip_myisam = $args['skip-myisam'];
+  if ($skip_myisam) {
+    $skip_myisam = TRUE;
```

Comment:
Wouldnt `$skip_myisam` always be already true at this statement?

Reply:
Sort of. It would be `1`, but I thought, an explicit `TRUE` is cleaner, as the default value if not set is an explicit `FALSE`.

Bad idea?

Reply:
I dont know TBH. Cant hurt to make it explicit, as you say.

Reply:
Ah, OK... I took a closer look and realized, that the conversion to bool already happens in function simpletest_script_parse_args(), so that's actually redundant. :+1: